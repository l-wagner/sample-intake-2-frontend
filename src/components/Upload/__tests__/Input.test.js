import React, { Component } from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import TestInput from '../Input'

const Input = props => <TestInput {...props} />

describe('Render Input', () => {
  it('render Input correctly', () => {
    const InputComponent = renderer.create(<Input />).toJSON()
    expect(InputComponent).toMatchSnapshot()
  })
})
