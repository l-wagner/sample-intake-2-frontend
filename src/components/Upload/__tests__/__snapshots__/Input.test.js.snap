// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Input render Input correctly 1`] = `
<div
  className="MuiFormControl-root-2 Input-textField-1"
>
  <label
    className="MuiFormLabel-root-17 MuiInputLabel-root-6 MuiInputLabel-formControl-11 MuiInputLabel-animated-14"
    data-shrink={false}
  >
    Missing translationId: upload.form.undefined_label for language: \${ languageCode }
  </label>
  <div
    className="MuiInputBase-root-37 MuiInput-root-24 MuiInput-underline-28 MuiInputBase-formControl-38 MuiInput-formControl-25"
    onClick={[Function]}
  >
    <input
      aria-invalid={false}
      className="MuiInputBase-input-47 MuiInput-input-32 MuiInputBase-inputType-50 MuiInput-inputType-35"
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      type="number"
    />
  </div>
  <p
    className="MuiFormHelperText-root-54"
  >
    Missing translationId: upload.form.undefined_helptext for language: \${ languageCode }
  </p>
</div>
`;
